Terminal-iOS/Study/CreateStudy/CreateStudyProtocols.swift

```diff
@@ -61,13 +63,15 @@ protocol CreateStudyPresenterProtocols: class {
     func showNotionValidResult(result: Bool)
     func showEvernoteValidResult(result: Bool)
     func showWebValidResult(result: Bool)
+    func studyInfoInvalid(message: String)
+    func studyInfoValid(message: String)
 }
 
 protocol CreateStudyRemoteDataManagerProtocols: class {
     func getNotionValid(id: String?) -> Bool
     func getEvernoteValid(url: String?) -> Bool
     func getWebValid(url: String?) -> Bool
-    func postStudy(study: StudyDetailPost) -> Bool
+    func postStudy(study: StudyDetailPost, completion: @escaping (_: Bool, _: String ) -> Void)
```

Comment:
```suggestion
    func postStudy(study: StudyDetailPost, completion: @escaping (result: Bool, message: String ) -> Void)
```
이런식으로 해주면 명확해서 보기 좋을거 같은데 어떻게 생각하나요?